src/main/java/lotto/domain/number/BonusNumber.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain.number;
+
+import static lotto.domain.number.LottoRandomNumber.LIMIT_RANGE;
+import static lotto.domain.number.LottoRandomNumber.START_RANGE;
+
+import lotto.domain.lotto.WinningLotto;
+import lotto.dto.request.BonusNumberRequest;
+
+public class BonusNumber {
+
+    private final int bonusNumber;
+
+    public BonusNumber(BonusNumberRequest bonusNumber, WinningLotto winningLotto) {
+        this.bonusNumber = validateType(bonusNumber);
+        validateRange();
+        winningLotto.validateIsDuplicate(this.bonusNumber);
+    }
+
+    private int validateType(BonusNumberRequest bonusNumber) {
+        try {
+            return Integer.parseInt(bonusNumber.getBonusNumber());
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException("[ERROR] 잘못된 타입 입니다.");
+        }
+    }
+
+    private void validateRange() {
+        if (bonusNumber < START_RANGE || LIMIT_RANGE < bonusNumber) {
```

Comment:
이처럼 두 가지 조건 이상이 논리연산자로 결합된 경우에는 해당 조건 판별을 메서드로 추출하여 가독성을 높일 수 있어요!
```suggestion
        if (isNumberInRange(bonusNumber)) {
```